Libraries/ConfigParser/inc/structs.h

```diff
@@ -0,0 +1,118 @@
+#pragma once
+
+#include <stdint.h>
+#include <string>
+
+namespace StreamingToolkit
+{
+	/* 
+	 * Server app configuration
+	 */
+	typedef struct
+	{
+		/* The video frame width						*/
```

Comment:
Yeah, it's because I love the tail alignment as this one:
https://dxr.mozilla.org/mozilla-beta/source/media/webrtc/trunk/webrtc/modules/audio_coding/codecs/isac/main/source/structs.h